modules/nodes-custom-scheduler-deploying.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-custom-scheduler.adoc
+
+[id="nodes-custom-scheduler-packaging_{context}"]
+= Deploying a custom scheduler
+
+To include a custom scheduler in your cluster, include the image for a custom scheduler in a deployment.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created a scheduler binary.
++
+[NOTE]
+====
+Information on how to create the scheduler binary is outside the scope of this document. For an example, see link:https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers[Configure Multiple Schedulers] in the Kubernetes documentation.
+====
++
+// TODO: update this statement. see github comments
+* You have created an image containing the scheduler binary and pushed it to a registry.
+
+.Procedure
+
+. Create a file that contains the deployment resources for the custom scheduler:
++
+.Example `custom-scheduler.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: custom-scheduler
+  namespace: kube-system <1>
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: custom-scheduler-as-kube-scheduler
+subjects:
+- kind: ServiceAccount
+  name: custom-scheduler
+  namespace: kube-system <1>
+roleRef:
+  kind: ClusterRole
+  name: system:kube-scheduler
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: custom-scheduler
+  namespace: kube-system <1>
+spec:
+  selector:
+    matchLabels:
+      component: scheduler
+      tier: control-plane
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      serviceAccountName: custom-scheduler
+      containers:
+      - command:
+        - /usr/local/bin/kube-scheduler
```

Comment:
Users can mount the config as a configmap, or as a local file on the host. This would be good to mention, though I would assume that developers who are at the point of deploying a custom scheduler would already be familiar enough with the platform to know that they can pass this as a mounted volume to the `--config` argument.

Reply:
@damemi @bergerhoffer As we discussed, we can put an example section (can be a separate .adoc file) to show how to deploy a load-aware scheduler following this tutorial. I drafted the section here: https://docs.google.com/document/d/1Yg3d0a0WY5L2tOV4N6PwtsX7OBFvx2oBky5JICCTQ44/edit?usp=sharing

Please feel free to revise it if needed.

Reply:
These docs don't need an example for deploying the load-aware scheduler, that is too specific for what we are trying to cover in this section. The load-aware scheduler config is documented upstream and we will also have blog posts on it; this should cover generic scheduler deployment steps.